english/coffee/coffee.md

```diff
@@ -31,6 +31,6 @@ After having your coffee do you want to:
 
 [Bake yourself something sweet to go with this delicious brew?](cookie/bake-cookies.md)
 
-
+[Eat a Bagel?](../bagel/eat-a-bagel.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.